### PR TITLE
Send failures to slack.

### PR DESCRIPTION
**Pull request by hibooboo2**

### CODE REVIEW

File: scripts/validation-tests [hunk 1]
```diff
@@ -42,6 +42,18 @@ notify_alert()
     MESSAGE=${message} ../scripts/slack-notification
 }
 
+send_failures()
+{
+    LOG_FILE="/opt/cattle/validation/logs/validationTestsJunit.xml"
```
Comment (cloudnautique) [inline scripts/validation-tests]: Should this use a PREFIX that is common with the bind mount?

File: scripts/validation-tests [hunk 2]
```diff
@@ -67,11 +79,13 @@ run_tests()
         -e "AMAZON_SECRETKEY=${AMAZON_SECRETKEY}" \
         -e "AMAZON_ACCESSKEY=${AMAZON_ACCESSKEY}" \
         -v $(pwd):/source \
+        -v /opt/cattle/validation/:/target \
```
Comment (cloudnautique) [inline scripts/validation-tests]: Where does /opt/cattle/validation come from? Does it need to be present to bind mount it in?

Comment (hibooboo2) [inline scripts/validation-tests]: I believe docker creates the directory you bind mount to if it doesn't exist.

File: scripts/validation-tests (hunk 1 above)
Comment (hibooboo2) [inline scripts/validation-tests]: Ok.

File: scripts/validation-tests (hunk 2 above)
Comment (cloudnautique) [inline scripts/validation-tests]: Indeed you are correct.

**Comment by cloudnautique** [comment]
Let me know when they have run. I'll LGTM and merge.

**Comment by cloudnautique** [comment]
Is this updated with the latest formatting?

**Pull request merged**